File: Latihan B/JAVA/Ram.java
public class Ram 
{
    private int capacity;
    private int price;

    // konstruktor
    public Ram(){}
    public Ram(int capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    // set untuk semua variable
    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    // get untuk semua variable
    public int getCapacity() {
        return capacity;
    }
    public int getPrice() {
        return price;
    }

    // tampil untuk Ram
    public void printRam() {
        System.out.println("  Kapasitas Ram      : " + capacity);
        System.out.println("  Harga Ram          : " + price);
    }
}

File: Latihan B/PHP/Ram.php
<?php
class Ram {
    private $capacity;
    private $price;

    // kosnturktor
    function __construct($capacity, $price) {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // set untuk semua variable
    function setCapacity($capacity) {
        $this->capacity = $capacity;
    }
    function setPrice($price) {
        $this->price = $price;
    }

    // get untuk semua variable
    function getCapacity() {
        return $this->capacity;
    }
    function getPrice() {
        return $this->price;
    }

    // tampil untuk ram
    function printRam() {
        echo "Kapasitas Ram : ". $this->getCapacity(). "<br/>";
        echo "Harga Ram : ". "Rp. ". $this->getPrice(). "<br/>";
    }

    // destructor
    function __destruct() {}
}
?>